next.config.js

```diff
@@ -4,7 +4,7 @@ const nextConfig = {
   // Configure `pageExtensions` to include MDX files
   pageExtensions: ['js', 'jsx', 'mdx', 'ts', 'tsx'],
   // Optionally, add any other Next.js config below
-  output: 'export',
+  //output: 'export',
```

Comment:
```suggestion
  ...(process.env.NODE_ENV === 'production' && {
    output: 'export',
  }),
```

브랜치마다 다르게 하는 것보다 env로 처리하는게 나을 것 같습니다 !!